posts/2021-08-30-open-liberty-stack-gradle.adoc

```diff
@@ -0,0 +1,96 @@
+
+== Open Liberty devfile stack
+
+The link:https://github.com/OpenLiberty/application-stack#open-liberty-application-stack[Open Liberty devfile stack] provides much of the infrastructure (Open Liberty, Maven/Gradle, Open J9, etc.) needed to start developing applications that use Maven or Gradle, and it is made available as Maven and Gradle development images. These images are used by the devfiles provided by the stack to serve as the base for building and running your applications. 
+
+The Open Liberty devfile stack provides two fully configured link:https://docs.devfile.io/devfile/2.1.0/user-guide/index.html[devfiles]: A link:https://github.com/devfile/registry/blob/main/stacks/java-openliberty/devfile.yaml[Maven based devfile] and a link:https://github.com/devfile/registry/blob/main/stacks/java-openliberty-gradle/devfile.yaml[Gradle based devfile]. These devfiles will allow you to deploy your Maven or Gradle built applications with ease as these devfiles define the environment and set of steps needed to build and deploy your application using the Open Liberty runtime.
+
+== OpenShift Do (odo)
+
+link:https://odo.dev[Odo] is a simple CLI tool that you use to create devfile based components that interact directly with your Kubernetes cluster to setup the environment, and build/deploy/access/debug your application. Directives to manage the environment and application are provided by a component's devfile.
+
+== Try it out
+
+To begin, you need three things:
+
+- Kubernetes cluster (i.e. Openshift, Minikube). Be sure to log in. Odo will interact with your kubernetes cluster.
+- Openshift Do. If you have not already installed odo, do so now by following the instructions outlined in the link:https://odo.dev[odo] documentation.
+- An application using Gradle. You can use your own application, or use one of the stack provided applications such as the link:https://github.com/OpenLiberty/application-stack-intro[intro] or link:https://github.com/OpenLiberty/application-stack-starters[starter] application. Instruction on how to use the starter application can be found link:https://github.com/OpenLiberty/application-stack/wiki/Using-the-Default-Starter#getting-started[here]. For this blog, we will use the intro application. This is a basic application that uses JAX-RS and MicroProfile technologies.
+
+Clone the application.
+
+[source,sh]
+----
+$ git clone git@github.com:OpenLiberty/application-stack-intro.git && \
+  cd application-stack-intro
+----
+
+Create an odo component of type java-openliberty-gradle. When the component is created, the Gradle based devfile provided by the Open Liberty devfile stack is downloaded to the root of the application.
+
+[source,sh]
+----
+$ odo create java-openliberty-gradle my-ol-component
+----
+
+Deploy the application on your Kubernetes cluster.
+
+[source,sh]
+----
+$ odo push
+----
+
+That is all. The application is now deployed on your cluster, and a URL has been made available to access the application. 
```

Comment:
suggested edit:
The application is now deployed on your cluster, and a URL has been made available to access the application. 
--->
The application is now deployed on your cluster and a URL is available to access it.